jwql/website/apps/jwql/data_containers.py

```diff
@@ -32,7 +32,6 @@
 import tempfile
 
 from astropy.io import fits
-from astropy.table import Table
```

Comment:
Were we not using `Table` in this module @BradleySappington?

Reply:
Correct, my linter told me it was imported and unused.  Doing minor code cleanups along the way.  There are more cleanups/errors/bugs in this file that need to be done, but they affect the logic directly so I didn't want to put them in this code request, they should be done separately.